<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../pictures/appIcon.png</normaloff>../pictures/appIcon.png</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="inputPart" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#inputPart{
background-color:qlineargradient(spread:pad, x1:0.125, y1:0.238636, x2:1, y2:0.625, stop:0.448864 rgba(255, 216, 196, 255), stop:0.772727 rgba(190, 255, 178, 255));
}</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 20pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Solution</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="guessHolder" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>431</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-color: rgb(85, 83, 58);
border:2.2px solid;
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QScrollArea" name="guessesArea">
       <property name="styleSheet">
        <string notr="true">#guessesArea{
background-color: rgb(255, 249, 217);
}</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="justGuesses">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>415</width>
          <height>155</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 20pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Step by Step</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#previousButton{
	border: 1.3px solid #ccc;
	border-radius:5px; 
	font: 9pt &quot;MS Shell Dlg 2&quot;;
	
	background-color: rgb(90, 90, 90);
}

#previousButton:hover{
	background-color:#584082;
}
</string>
    </property>
    <property name="text">
     <string>Previous page</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../pictures/backArrow.png</normaloff>../pictures/backArrow.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="noSolutionLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>801</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid #262624; 
font: 20pt &quot;Arial&quot;;
background-color:#686968;
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>The system is singular so it does not have a unique solution</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid #262624; 
font: 9pt &quot;Arial&quot;;
background-color:#686968;
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>381</y>
      <width>831</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTableWidget {
                background-color: #f5f5f5;
                alternate-background-color: #e0e0e0;
                selection-background-color: #0078d7;
                selection-color: white;
                border: 1px solid #ccc;
            }

            QTableWidget QHeaderView::section {
                background-color: #0078d7;
                color: white;
                padding: 4px;
                border: 1px solid #ccc;
            }

            QTableWidget::item:selected {
                background-color: #0078d7;
                color: white;
            }</string>
    </property>
   </widget>
   <widget class="QLabel" name="diverge">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid #262624; 
font: 14pt &quot;Arial&quot;;
background-color:#686968;
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>System diverged</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
